app/routes/docs.faq.md

```diff
@@ -66,6 +66,30 @@ This warning means that there have not been any recent alerts on that topic.
 
 To get started, [sign in or sign up](https://gcn.nasa.gov/login) and then select 'Email Notifications' from account dropdown menu. See also [GCN Circular 32517](https://gcn.gsfc.nasa.gov/gcn3/32517.gcn3).
 
+## Notice Producers
+
+### Which unit system is recommended to be used?
```

Comment:
Eliminate passive voice.

Reply:
This will be moved from FAQ to tutorial page.